specification/digitaltwins/resource-manager/Microsoft.DigitalTwins/preview/2020-03-01-preview/digitaltwins.json

```diff
@@ -512,171 +512,6 @@
         }
       }
     },
-    "/{scope}/providers/Microsoft.DigitalTwins/integrationResources/{integrationResourceName}": {
```

Comment:
removing a bunch of APIs - breaking change for preview customers. Their existing scenarios will be broken.
Do we have data on the usage of these APIs? OK to break them?